packages/exoflex/example/metro.config.js

```diff
@@ -14,6 +14,9 @@ let extraNodeModules = {};
   'react-native-svg',
   'expo-font',
   '@babel/runtime',
+  // We probably shouldn't use the entire lodash package here.
+  // TODO: Remove direct import from lodash in exoflex.
+  'lodash',
```

Comment:
Is this because of `lodash.mergeWith` from exoflex's package.json?

Reply:
No, this is because we imported `lodash` package directly inside `Switch.tsx` without specifying it in our package.json as dependency.

Because of how our example project configured, it'll only allow importing from specified list of dependencies instead of all packages installed in `node_modules`.